bbtools/38.86/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:xenial
+
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="BBTools"
+LABEL software.version="38.86"
+LABEL description="A set of tools labeled as \"Bestus Bioinformaticus\""
+LABEL website="https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/"
+LABEL license="https://jgi.doe.gov/disclaimer/"
+LABEL maintainer="Abigail Shockey"
+LABEL maintainer.email="abigail.shockey@slh.wisc.edu"
+
+RUN apt-get update && apt-get install -y build-essential \
+  openjdk-8-jre-headless \
+  wget \
+  samtools
+
+# get miniconda and the artic-ncov19 repo
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+  bash ./Miniconda3-latest-Linux-x86_64.sh -p /miniconda -b  && \
+  rm Miniconda3-latest-Linux-x86_64.sh
+
+# set the environment
+ENV PATH="/miniconda/bin:$PATH"
+
+RUN conda config --add channels defaults && \
+  conda config --add channels bioconda && \
+  conda config --add channels conda-forge && \
+  conda install sambamba
+
+RUN wget https://sourceforge.net/projects/bbmap/files/BBMap_38.86.tar.gz && \
+  tar -xzf BBMap_38.86.tar.gz && \
+  rm BBMap_38.86.tar.gz
+
+ENV PATH="${PATH}:/bbmap"
```

Comment:
Might be worth adding in LC_ALL here too, for singularity compatibility
```Dockerfile
ENV PATH="${PATH}:/bbmap"\
 LC_ALL=C
```

Reply:
And this

Reply:
ty!